src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts

```diff
@@ -326,6 +341,23 @@ export class JJCountComponent implements OnInit {
       this.form.get('revisedDueDate').setValue(this.jjDisputedCount?.dueDate);
     }
   }
+
+  //Latest Plea
+  bindLatestPlea(value){
+    this.form.get("latestPlea").setValue(value);
```

Comment:
Can it be typed? `this.form.controls.latestPlea?.setValue(value)`

Reply:
This particular page have the same pattern of assigning values to control, so went with this pattern. I can change it , what do you think?

Reply:
I think the original pattern was created before Angular 14. After Angular 14 updates, I think we should change it to new pattern to keep those coding being typed, so we can know the errors before build. Just use the new pattern for any new coding first, we change the old codings later.